src/layout-grid/types.js

```diff
@@ -62,6 +66,8 @@ export type CellPropsT = {
   skip?: ResponsiveT<number>,
   /** Control number of columns the cell should span. */
   span?: ResponsiveT<number>,
+  /** COntrol CSS units */
+  gridUnit?: string,
```

Comment:
Said above, prob shouldn't have this as a top level prop for Cell.

Reply:
noted! I have removed this.